Packages/Status/Sources/Status/Editor/PollView.swift

```diff
@@ -0,0 +1,128 @@
+import SwiftUI
+import DesignSystem
+import Env
+
+struct PollView: View {
```

Comment:
Should this be named StatusEditorPollView for clarity, and be filed within the `Components` folder as the other accessories?

Reply:
This sounds good to me. I will update this too!